samples/betaFeatures.js

```diff
@@ -0,0 +1,610 @@
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+
+  //[END speech_transcribe_multilang]
+}
+
+function speechTranscribeMultilangGCS(gcsUri) {
+  // [START speech_transcribe_multilang_gcs]
+
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const uri = path to GCS audio file e.g. `gs:/bucket/audio.wav`;
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+    audioChannelCount: 2,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .longRunningRecognize(request)
+    .then(data => {
+      const operation = data[0];
+      // Get a Promise representation of the final result of the job
+      return operation.promise();
+    })
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_multilang_gcs]
+}
+
+// WORD LEVEL CONFIDENCE
+function speechTranscribeWordLevelConfidence(speechFile) {
+  // [START speech_transcribe_word_level_confidence]
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const fileName = 'Local path tols -a audio file, e.g. /path/to/audio.raw';
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordTimeOffsets: true,
+    enableWordConfidence: true,
+  };
+
+  const audio = {
+    content: fs.readFileSync(speechFile).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        ` Transcription: ${transcription} \n Confidence: ${confidence}`
```

Comment:
If I'm reading this correctly, this will print out each transcript on a separate line, then print out their confidences in the same order. This seems weird, no? I'd expect the confidence to be close to the transcript it corresponds to. Perhaps:

```js
transcription.forEach((transcript, i) => console.log(`Transcription: ${confidence[i]}: ${transcript}`))
```
?